### ISSUE TITLE
SPEC-MTP annotations different than original annotations

### ISSUE THREAD

**Issue by pixelite1201** [WenjiaWang0312/Zolly#4]
Hello,

I tried downloading the SPEC-MTP annotation from OneDrive. It seems that the focal length in the annotation file is different than the focal length used in the annotation file downloaded from [SPEC website](https://spec.is.tue.mpg.de/). Did you do some post processing to the images?

**Comment by WenjiaWang0312** [comment]
Hi there. I looked at each image carefully by my eyes, and found many of their annotations were not precise. I filtered out about 80 images with ridiculous betas or pose parameters. And I also optimized the focal length by re-projection loss and made sure every image was better overlayed than theirs. During testing, I reported these methods all on my modified version, so the experiments are fair. The pose and betas were not modified so the 3D metrics are same. The focal length only affects the mIoU and p-mIoU.
